manifests/params.pp

```diff
@@ -18,15 +17,43 @@
 
   case $::osfamily {
     'Debian': {
+      $jenkins_user     = 'jenkins'
+      $jenkins_group    = 'jenkins'
+      $repo             = true
+      $homedir          = '/var/lib/jenkins'
+      $jenkinsdir       = $homedir
+      $default_plugin_dir = "${jenkinsdir}/plugins"
```

Comment:
Might be worth lining up the indenting even though the linter won't complain about it.